imports/plugins/core/orders/client/containers/ordersListContainer.js

```diff
@@ -140,30 +137,263 @@ class OrdersListContainer extends Component {
     return false;
   }
 
-  render() {
-    const { handleShowMoreClick } = this.props;
+  /**
+   * shippingStatusUpdateCall
+   *
+   * @summary set selected order(s) to the provided shipping state
+   * @param {Array} selectedOrders - array of selected orders
+   * @param {String} status - the shipping status to be set
+   * @return {null} no return value
+   */
+  shippingStatusUpdateCall = (selectedOrders, status) => {
+    let orderText;
+
+    if (selectedOrders.length > 1) {
+      orderText = "orders";
+    } else {
+      orderText = "order";
```

Comment:
I'm passing this variable to `i18next.t` below to be translated depending on context.